pcdscalc/constants.py

```diff
@@ -0,0 +1,345 @@
+    "Ne": 0.0008999,
+    "Na": 0.971,
+    "Mg": 1.738,
+    "Al": 2.698,
+    "Si": 2.3296,
+    "P": 1.82,
+    "S": 2.067,
+    "Cl": 0.003214,
+    "Ar": 0.0017837,
+    "K": 0.862,
+    "Ca": 1.54,
+    "Sc": 2.989,
+    "Ti": 4.54,
+    "V": 6.11,
+    "Cr": 7.15,
+    "Mn": 7.44,
+    "Fe": 7.874,
+    "Co": 8.86,
+    "Ni": 8.912,
+    "Cu": 8.96,
+    "Zn": 7.134,
+    "Ga": 5.907,
+    "Ge": 5.323,
+    "As": 5.776,
+    "Se": 4.809,
+    "Br": 3.122,
+    "Kr": 0.003733,
+    "Rb": 1.532,
+    "Sr": 2.64,
+    "Y": 4.469,
+    "Zr": 6.506,
+    "Nb": 8.57,
+    "Mo": 10.22,
+    "Tc": 11.5,
+    "Ru": 12.37,
+    "Rh": 12.41,
+    "Pd": 12.02,
+    "Ag": 10.501,
+    "Cd": 8.69,
+    "In": 7.31,
+    "Sn": 7.287,
+    "Sb": 6.685,
+    "Te": 6.232,
+    "I": 4.93,
+    "Xe": 0.005887,
+    "Cs": 1.873,
+    "Ba": 3.594,
+    "La": 6.145,
+    "Ce": 6.77,
+    "Pr": 6.773,
+    "Nd": 7.007,
+    "Pm": 7.26,
+    "Sm": 7.52,
+    "Eu": 5.243,
+    "Gd": 7.895,
+    "Tb": 8.229,
+    "Dy": 8.55,
+    "Ho": 8.795,
+    "Er": 9.066,
+    "Tm": 9.321,
+    "Yb": 6.965,
+    "Lu": 9.84,
+    "Hf": 13.31,
+    "Ta": 16.654,
+    "W": 19.25,
+    "WC": 15.8,
+    "Re": 21.02,
+    "Os": 22.61,
+    "Ir": 22.56,
+    "Pt": 21.46,
+    "Au": 19.282,
+    "Hg": 13.5336,
+    "Tl": 11.85,
+    "Pb": 11.342,
+    "Bi": 9.807,
+    "Po": 9.32,
+    "At": 7,
+    "Rn": 0.00973,
+    "Fr": 1.87,
+    "Ra": 5.5,
+    "Ac": 10.07,
+    "Th": 11.72,
+    "Pa": 15.37,
+    "U": 18.95,
+    "Np": 20.45,
+    "Pu": 19.84,
+    "H2O": 1.0,
+    "B4C": 2.52,
+    "SiC": 3.217,
+    "SiO2": 2.2,
+    "Al2O3": 3.97,
+    "ZnSe": 5.42,
+    "ZnTe": 6.34,
+    "CdS": 6.749,
+    "CdSe": 7.01,
+    "CdTe": 7.47,
+    "BN": 3.49,
+    "GaSb": 5.619,
+    "GaAs": 5.316,
+    "GaMnAs": 5.316,
+    "GaP": 4.13,
+    "InP": 4.787,
+    "InAs": 5.66,
+    "InSb": 5.775,
+    "TaC": 13.9,
+    "TiB2": 4.52,
+    "YAG": 4.55,
+    "CuBe": 8.96,
+    "ZnO": 5.606,
+    "SiC2": 3.217,
+    "AlN": 3.3,
+    "Si3N4": 3.44,
+    "CaF2": 3.18,
+    "LiF": 2.635,
+    "KF": 2.48,
+    "PbF2": 8.24,
+    "SrF2": 4.24,
+    "KBr": 2.75,
+    "ZrO2": 5.6,
+    "Gd3Ga5O12": 7.08,
+    "CaSiO5": 2.4,
+    "LaMnO3": 5.7,
+    "LaAlO3": 6.52,
+    "La0.7Sr0.3MnO3": 6.17,
+    "La0.5Ca0.5MnO3": 6.3,
+    "Fe.68Cr.2Ni.1Mn.02": 8.03,
+    "CaSO4H4O2": 2.32,
+    "C10H8O4": 1.4,
+    "C22H10N2O5": 1.43,
+    "C3H6O": 0.79,
+    "C5H8O2": 1.19,
+    "C2F4": 2.2,
+    "C7H8": 0.867,
+    "Y3Al5O12": 4.56,
+    "CHN.3O7.6": 1.06,
+    "C1.5H0.3O4.3N0.4PCa2.2": 1.92,
+    ("Be0.9983O0.0003Al0.0001Ca0.0002C0.0003Cr0.000035Co0.000005Cu0.00005Fe0."
+     "0003Pb0.000005Mg0.00006Mn0.00003Mo0.00001Ni0.0002Si0.0001Ag0.000005Ti0."
+     "00001Zn0.0001"): 1.85,
+    ("Be.994O.004Al.0005B.000003Cd.0000002Ca.0001C.0006Cr.0001Co.00001Cu."
+     "0001Fe.0008Pb.00002Li.000003Mg.00049Mn.0001Mo.00002Ni.0002N.0003Si."
+     "0004Ag.00001"): 1.85,
+}
+
+
+# Chemical Formula Aliases
+alias = {
```

Comment:
Since `constants` can be shared in the future, it'd be better to be specific for what these alias:
```suggestion
chemical_name_to_formula = {
```